docs/03-tutorials/00-observability/README.md

```diff
@@ -4,22 +4,22 @@ title: Observability - tutorials
 
 ## Purpose
 
-The following instructions describe the complete monitoring flow for your services in Kyma. You get the gist of Kyma built-in monitoring applications, such as Prometheus, Grafana, and Alertmanager. You will learn how and where you can observe and visualize your service metrics to monitor them for any alerting values.
+The following instructions describe the complete monitoring flow for your services in Kyma. You get the gist of monitoring applications, such as Prometheus, Grafana, and Alertmanager. You will learn how and where you can observe and visualize your service metrics to monitor them for any alerting values.
```

Comment:
According to the content guidelines the present tense should be used rather than the future form.